VersionControlUI/src/DiffManager.cpp

```diff
@@ -57,25 +57,22 @@ namespace VersionControlUI
 
 const QString DiffManager::OVERVIEW_HIGHLIGHT_OVERLAY_NAME = "changeOverviewHighlights";
 const QString DiffManager::OVERVIEW_ICON_OVERLAY_NAME = "changeOverviewIcons";
-
+const QString DiffManager::NAME_CHANGE_OVERLAY_NAME = "nameChange_overlay";
+const QString DiffManager::NAME_CHANGE_ARROW_OVERLAY_NAME = "nameChange_arrow_overlay";
+QHash<Model::NodeIdType, bool> DiffManager::nameChangesIdsIsNameText_;
+QList<ChangeWithNodes> DiffManager::nameChanges_;
+DiffManager::NameChangeVisualizations DiffManager::nameChangeVisualization_{Summary};
+QList<int> DiffManager::nameChangeOnZoomHandlerIds_;
 QHash<Visualization::ViewItem*, int> DiffManager::onZoomHandlerIdPerViewItem_;
-
-struct VersionNodes {
-	Model::Node* oldNode_{};
-	Model::Node* newNode_{};
-};
-
-struct ChangeWithNodes {
-	Model::NodeIdType id_;
-	VersionNodes versionNodes_;
-	FilePersistence::ChangeType changeType_{FilePersistence::ChangeType::Unclassified};
-};
+QSet<Visualization::Item*> DiffManager::nameChangesScaledByAncestor_;
 
 DiffManager::DiffManager(QString project, QList<Model::SymbolMatcher> contextUnitMatcherPriorityList,
 								 Model::NodeIdType targetNodeID, NameChangeVisualizations nameChangeVisualization)
-	: project_{project}, contextUnitMatcherPriorityList_{contextUnitMatcherPriorityList}, targetNodeId_{targetNodeID},
-	  nameChangeVisualization_{nameChangeVisualization}
-{}
+	: project_{project}, contextUnitMatcherPriorityList_{contextUnitMatcherPriorityList}, targetNodeId_{targetNodeID}
+
+{
+	nameChangeVisualization_ = nameChangeVisualization;
```

Comment:
please, use the same multiplicity for both the type name and the variable names. It's kind of strange for something of type `Visualizations` to be called `visualization`.

Also this should be initialized above, like the rest of the fields, after the `:`.

Also, I think the name should be a little more explanatory, include `flags` at the end.